### PR TITLE
New recipe for pika

### PR DESCRIPTION
**Pull request by frol**
[Pika](https://pika.readthedocs.org/) is a pure-Python implementation of the AMQP 0-9-1 protocol that tries to stay fairly independent of the underlying network support library.

### CODE REVIEW

File: recipes/pika/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  version: {{ version }}
+
+source:
+  fn: pika-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pika/pika-{{ version }}.tar.gz
+  sha256: 7277b4d12a99efa4058782614d84138983f9f89d690bdfcea66290d810806459
+
+build:
+  number: 0
+  script: python setup.py install
```
Comment (jakirkham) [inline recipes/pika/meta.yaml]: As this uses `setuptools`, we need these arguments ( ` --single-version-externally-managed --record record.txt` ) as well.

File: recipes/pika/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  home: http://pika.readthedocs.org
+  license: BSD 3-clause
+  summary: Pure Python RabbitMQ/AMQP 0-9-1 client library
+
+extra:
+  recipe-maintainers:
+    - frol
+    - mgr
+    - tonyg
+    - vitaly-krugl
```
Comment (jakirkham) [inline recipes/pika/meta.yaml]: @mgr @tonyg @vitaly-krugl, are all of you ok with being added as maintainers on this recipe?

**Comment by jakirkham** [comment]
Might be of interest to you, @scopatz.

File: recipes/pika/meta.yaml (hunk 2 above)
Comment (frol) [inline recipes/pika/meta.yaml]: @jakirkham They are listed in [Pika Github organization](https://github.com/orgs/pika/people)

File: recipes/pika/meta.yaml (hunk 1 above)
Comment (frol) [inline recipes/pika/meta.yaml]: Fixed. CI is already happy.

File: recipes/pika/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/pika/meta.yaml]: Sure, but I would like to get their sign off on being added. Sometimes people don't want to be added as maintainers even though they are developers on the project.

Comment (frol) [inline recipes/pika/meta.yaml]: @gmr Would you like to be a maintainer of Pika recipe (package) for [Conda-Forge repository](https://conda-forge.github.io/)?

P.S. @jakirkham, I have typoed in mgr nick, it should have been @gmr.

**Comment by frol** [comment]
@jakirkham OK, since no response from the package developers so far, I have removed them from the list of this recipe maintainers. Can we merge now?

**Comment by frol** [comment]
@jakirkham Ping?

**Comment by jakirkham** [comment]
LGTM. Thanks @frol.

Just as FYI, I'm going to be in meetings most of the day. Maybe @ocefpaf can help you with any pressing recipes that you need in.

**Comment by ocefpaf** [comment]
> Maybe @ocefpaf can help you with any pressing recipes that you need in.

@frol it has been hard to follow the high traffic here at conda-forge, but feel free to ping me (and keep pinging) if you need eyes in a PR.

**Comment by frol** [comment]
@jakirkham @ocefpaf Thank you for keeping this project up and running! I'm sorry if I ping you too much.

**Comment by ocefpaf** [comment]
>  I'm sorry if I ping you too much.

Don't be! Just don't be mad if some slip under the radar. And try again :wink:

**Comment by scopatz** [comment]
Thanks for letting me know @jakirkham :)
